File: lihua-vue/src/api/monitor/server/type/MemoryMonitor.ts
export interface MemoryMonitor {
    /** 总内存大小 */
    total: string;

    /** 已使用大小 */
    used: string;

    /** 空闲大小 */
    available: string;

    /** 使用率 */
    usagePercentage: string;
}

File: lihua-cloud/lihua-biz/lihua-monitor/src/main/java/com/lihua/monitor/model/MemoryMonitor.java
package com.lihua.monitor.model;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class MemoryMonitor {

    /**
     * 总内存大小
     */
    private String total;

    /**
     * 已使用大小
     */
    private String used;

    /**
     * 空闲大小
     */
    private String available;

    /**
     * 使用率
     */
    private String usagePercentage;

}
